datadog-crashtracker/src/collector/emitters.rs

```diff
@@ -148,6 +152,11 @@ pub(crate) fn emit_crashreport(
         let fault_rsp = extract_rsp(ucontext);
         unsafe { emit_backtrace_by_frames(pipe, config.resolve_frames(), fault_rsp)? };
     }
+
+    if is_runtime_callback_registered() {
+        emit_runtime_stack(pipe)?;
+    }
```

Comment:
good point to send it once we've done every thing else (in case the callback is not signal-safe and block the thread)

When the feature is available, we would encourage libraries to add a flag so this feature is disable if an issue occurs (avoiding releasing a new version)